cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/JsonMenu/utilities.ts

```diff
@@ -47,9 +51,25 @@ function getJSONConfig(widgetJSONData) {
     ...(outputName && { name: outputName }),
   };
 
+  const filtersData = filters.map((filterID) => {
+    const filterToShowListData = filterToShowList[filterID].map((showID) => {
+      return {
+        name: showToInfo[showID].name,
+        ...(showToInfo[showID].type && {
```

Comment:
Could you please extract this out as an `if` condition? IMHO it is easier to read.

Reply:
Resolved